service/src/LevelZero.cpp

```diff
@@ -398,12 +398,19 @@ namespace geopm
         return frequency_status_helper(l0_device_idx, l0_domain, l0_domain_idx).actual;
     }
 
+    double LevelZeroImp::frequency_efficient(unsigned int l0_device_idx,
+                                             int l0_domain, int l0_domain_idx) const
+    {
+        return frequency_status_helper(l0_device_idx, l0_domain, l0_domain_idx).efficient;
```

Comment:
So I'm seeing these changes are simply exposing this efficient value that has been here all along. So how was it available before and what has changed that now you can expose it?

Reply:
LevelZero exposed the efficient frequency reported by the underlying FW/HW.  The FW/HW implementation may change depending on SKU, family, etc.  Exposing it now is based upon testing on recent hardware and identifying a use case for the value #2288